spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -150,13 +158,33 @@ def get_value(self, name):
 
     def set_value(self, name, value):
         """Set value for a variable"""
+        reason_mismatched_numpy = _(
+            "There is a mismatch between the Numpy versions used by Spyder "
+            "and the kernel of your current console. To fix this problem, "
+            "please upgrade `numpy` in the console environment to version 2."
```

Comment:
```suggestion
            "please upgrade <tt>numpy</tt> in the console environment to "
            "version 2.0 or higher."
```

- Same as above for `numpy`.
- Be more explicit about the required version in this case.